lib/ts/recipe/session/index.ts

```diff
@@ -44,6 +44,7 @@ export default class SessionWrapper {
         userId: string,
         accessTokenPayload: any = {},
         sessionData: any = {},
```

Comment:
need to add getSession / verifySession that do not depend on the req / res (our previous discussion about this)

Reply:
That'll be done in a separate PR

Reply:
https://github.com/supertokens/supertokens-node/pull/520